pkg/csi/service/wcp/controller.go

```diff
@@ -555,12 +555,16 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 	resp, faultType, err := createVolumeInternal()
 	log := logger.GetLogger(ctx)
 	log.Debugf("createVolumeInternal: returns fault %q", faultType)
+
+	namespace := common.GetNamespaceFromContext(ctx)
+	log.Debugf("Namespace from context metadata: %s", namespace)
```

Comment:
If namespace is unknown, can we add a warning log message? Same comment in all places.

Reply:
Done.